### DISCUSSION TITLE
240313

### DISCUSSION THREAD

**Discussion by jisunipark** [codeit-part4-8team-project/main#36] [팀회의]
## 멘토님 질문 사항

**Comment by sobokki** [comment]
## 논의사항
## 어제 한 일
캘린더 UI 수정 공휴일 api 연동 시도... (완성 X)
## 오늘 할 일
주간, 월간 캘린더 합치기 , 캘린더 context로 관리해보기 
## 오늘의 일정 (코어 타임)
## 멘토님 질문 사항

**Comment by jisunipark** [comment]
## 논의사항
- pr 확인해주시면 감사하겠습니다~

## 어제 한 일
- 사이드바 ui 수정 및 추가
   - https://github.com/codeit-part4-8team-project/main/pull/35

## 오늘 할 일
- 사이드바 컨텍스트 관리
- 새로운 사이즈 맞추어 레이아웃 ui

## 오늘의 일정 (코어 타임)

## 멘토님 질문 사항

**Comment by caseV12** [comment]
## 논의사항
백엔드 채린님이랑 CORS 에러
## 어제 한 일
css 셋팅
로그인, 회원가입 UI 완
## 오늘 할 일
회원가입 UI 리팩터링 필요, 유저 기능 완성
## 오늘의 일정 (코어 타임)
없습니다
## 멘토님 질문 사항

**Comment by MoonPillGyeom** [comment]
## 논의사항
지금 막힌 부분이 스타일 컬러를 뽑아야하는데 16진수로 안 나오고 rgba로 나와서 고민중입니다
## 어제 한 일
x
## 오늘 할 일
string해결해야함
## 오늘의 일정(코어 타임)
## 잘문사항

**Comment by jisunipark** [comment]
- 지선: 사이드바 컨텍스트 대신 url로 관리해보기
